core/themes/bartik/css/style.css

```diff
@@ -1185,6 +1185,28 @@ ul.action-links li a {
   padding: 0;
 }
 
+ /* -------------- Hero blocks  -------------- */
+
+ .hero-block {
+   font-size: 250%;
+   position: relative;
+   padding: 0;
+ }
+ .hero-block.maintain-aspect-ratio:before {
```

Comment:
I'd say nix the entire "maintain-aspect-ratio" class and just do this on all hero blocks (as would happen right now anyway).